roles/supervisor/tasks/main.yml

```diff
@@ -4,7 +4,7 @@
   apt: pkg=supervisor state=present
   become: true
 
-- name: make sure supervisord is running
+- name: check if supervisord is running
```

Comment:
`make sure` was more accurate, since this command ensures supervisord is running if it wasn't before, it doesn't just check and report whether it is the case.